### PR TITLE
removed hardcodded error strings

### LINKED ISSUE #675: Feature Request
**Is your feature request related to a problem? Please describe.**
So many hard coded values are scattered around the code that will cause problems later.

**Describe the solution you'd like**
a global Constants file or even folder that  hold them and helps manage the code more easily.

**Additional context**
This was requested by the mentors on : #674

### PR DESCRIPTION
**Pull request by ebrahim354**
**What kind of change does this PR introduce?**
replaces hard coded error strings with constant variables.

**Issue Number:**

Fixes #675

### CODE REVIEW

**Review by JamaicanFriedChicken — COMMENTED** [review]
Hey @ebrahim354, the discussion around this change was to introduce a constants file to store the hardcoded values which is good. However, to actually introduce the change on a widescale in this PR can cause breaking changes elsewhere and will likely cause merge conflicts to the other participants. 

What needs to be done was just to create the constants file and to make the change related to that particular file so then we can ask other participants to make the change as well in their PRs which will introduce this change slowly to prevent breaking changes.

**Comment by taslangraham** [comment]
> Hey @ebrahim354, the discussion around this change was to introduce a constants file to store the hardcoded values which is good. However, to actually introduce the change on a widescale in this PR can cause breaking changes elsewhere and will likely cause merge conflicts to the other participants.
> 
> What needs to be done was just to create the constants file and to make the change related to that particular file so then we can ask other participants to make the change as well in their PRs which will introduce this change slowly to prevent breaking changes.

@ebrahim354, I agree with what @JamaicanFriedChicken  said. Please update PR to only include the constants.js

**Comment by ebrahim354** [comment]
Sure, I will change it.

**Comment by ebrahim354** [comment]
This should be good as a start.

File: lib/resolvers/functions/creatorCheck.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
 const { UnauthorizedError } = require('errors');
 const requestContext = require('talawa-request-context');
+const { USER_NOT_AUTHORIZED } = require('../../../constants');
 
 const creatorCheck = (context, org) => {
   const isCreator = String(org.creator) === context.userId;
   if (!isCreator) {
     throw new UnauthorizedError(
       process.env.NODE_ENV !== 'production'
```
Comment (JamaicanFriedChicken) [inline lib/resolvers/functions/creatorCheck.js:9]: hey @ebrahim354, you're almost there. You can also place the `'production'` string in the constants file as well

File: lib/resolvers/organization_mutations/removeOrganization.js [hunk 2]
```diff
@@ -6,13 +6,17 @@ const Organization = require('../../models/Organization');
...
+const {
+  USER_NOT_FOUND,
+  ORGANIZATION_NOT_FOUND,
+} = require('../../../constants');
 
 const removeOrganizaiton = async (parent, args, context) => {
   const user = await User.findOne({ _id: context.userId });
   if (!user) {
     throw new NotFoundError(
       process.env.NODE_ENV !== 'production'
```
Comment (JamaicanFriedChicken) [inline lib/resolvers/organization_mutations/removeOrganization.js:18]: this as well

File: lib/resolvers/organization_mutations/removeOrganization.js [hunk 3]
```diff
@@ -24,7 +28,7 @@ const removeOrganizaiton = async (parent, args, context) => {
   if (!org) {
     throw new NotFoundError(
       process.env.NODE_ENV !== 'production'
```
Comment (JamaicanFriedChicken) [inline lib/resolvers/organization_mutations/removeOrganization.js:30]: and here

**Review by JamaicanFriedChicken — CHANGES_REQUESTED** [review]
Almost there.

**Comment by ebrahim354** [comment]
What about these strings should I replace them as well ? 
`
  requestContext.translate('user.notAuthorized'),
      'user.notAuthorized',
      'userAuthorization'
`

**Comment by JamaicanFriedChicken** [comment]
> What about these strings should I replace them as well ? `requestContext.translate('user.notAuthorized'), 'user.notAuthorized', 'userAuthorization'`

Yes, you should replace them.

**Comment by ebrahim354** [comment]
It should be working now.

**Review by JamaicanFriedChicken — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]

**Pull request merged**